Frontend/src/app/styles/display.scss

```diff
@@ -1,13 +1,14 @@
 .container {
-    width: 100%;
-    margin: 0 auto;
+  margin: 0 auto;
+  padding: 5%;

```

Comment:
5% seems kinda low on my screen I was trying some values out and 2% seemed to work well. Let me know what you think it could look completely off on your monitor

Reply:
I agree, If you're talking about the padding at the top as I believe. I removed the top padding completely.